assemblies/installing-odh-v2.adoc

```diff
@@ -0,0 +1,33 @@
+:_module-type: ASSEMBLY
+//pv2hash: 
+
+ifdef::context[:parent-context: {context}]
+
+:context: installv2
+
+[id="installing-odh-v2_{context}"]
+= Installing Open Data Hub (v2)
+
+[role='_abstract']
+You can install the Open Data Hub Operator version 2 on your OpenShift Container Platform from the OpenShift web console. 
+//For information about upgrading the Open Hub Operator from version 1 to version 2, see
+
+Installing Open Data Hub involves the following tasks:
+
+. Install the Open Data Hub Operator.
+. Install Open Data Hub components.
+. Access the Open Data Hub dashboard.
+
+[NOTE]
+--
+Open Data Hub Operator version 2 represents an alpha release, only accessible on the *fast* channel. Later releases will change to the *rolling* channel when the operator attains greater stability.
```

Comment:
Suggest a few tweaks here: 

"Version 2  of the Open Data Hub Operator represents an alpha release, accessible only on the `fast` channel. Later releases will change to the `rolling` channel, when the Operator is more stable."